src/components/EventListPreview/index.js

```diff
@@ -0,0 +1,21 @@
+import StyledListItem from "../StyledListItem";
+import StyledDivider from "../StyledDivider";
+import shortenText from "../../utils/shortenText";
+
+export default function EventListPreview({ event }) {
+  return (
+    <>
+      <StyledListItem>
+        <time
+          dateTime={event.startDate}
+          aria-label="date"
+        >{`${event.startDate.slice(8, 10)}.${event.startDate.slice(
+          5,
+          7
+        )}.${event.startDate.slice(0, 4)}`}</time>
+        <h4 aria-label="title">{shortenText(event.title)}</h4>
```

Comment:
If you are using an aria label it should be more descriptive then the contents of the element. Just `"title"` is not. Since you want to use this for testing as well I'd suggest using the "unshortend" version of the event title here. Then you can also test for that.

```suggestion
        <h4 aria-label={event.title}>{shortenText(event.title)}</h4>
```